### DISCUSSION TITLE
First load experience for guest user

### DISCUSSION THREAD

**Discussion by jaril** [RecordReplay/devtools#5196] [General]
Moved from [4859](https://github.com/RecordReplay/devtools/issues/4859).

https://www.loom.com/share/fdf1039deca24a9e80dfb8d78994fd6d

The main points appear to be:
- the user is not told what replay is in the first place, or what they're looking at.
- sign in is confusing (for reasons documented in https://github.com/RecordReplay/devtools/issues/5065)
- loading screen should differentiate between logged in/not.
- Replay Info, is it important?

**Comment by jonbell-lot23** [comment]
March 15, 2022:

This seems like the sort of thing I can take on, especially as part of the onboarding audit.
